charts/moneypenny/values.yaml

```diff
@@ -44,15 +44,15 @@ ingress:
   enabled: false
   annotations:
     nginx.ingress.kubernetes.io/auth-method: GET
-    nginx.ingress.kubernetes.io/auth-url: "http://gafaelfawr-service.gafaelfawr.svc.cluster.local:8080/auth?scope=exec:admin"
+    nginx.ingress.kubernetes.io/auth-url: "http://gafaelfawr-service.gafaelfawr.svc.cluster.local:8080/auth?scope=admin:provision"
```

Comment:
I think I missed the provision thing in the list of scopes in the gafaelfawr chart?  It just doesn't ring a bell.  Maybe I missed it.

Reply:
You haven't seen it anywhere else yet -- it will be added in the phalanx diff as the scope to give the nublado2 token. Thanks for pointing this out; it reminded me that I forgot to pin the moneypenny and nublado2 charts. I was able to make the code backward-compatible, but not the charts, so upgrading the charts has to be done simultaneously with Gafaelfawr 2.0. Taking care of that now.